packages/migration/ecosysystem.go

```diff
@@ -466,6 +472,7 @@ MenuItem(
 			Conditions string
 			ValidateCount int "optional"
 			ApplicationId int "optional"
+			ValidateMode int "optional"
```

Comment:
why int instead of bool?

Reply:
We don't use type bool as column's type in PostgreSQL. So, in any case, we will need to convert it to 0 or 1.